00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
> Since it is a path, you basically prepend the "test environment" at the front. This way, you get to load "test fixtures" first and fallback to global ones otherwise.

Jep, but you have to create the very same folder structure (including subfolders), also when you could only need to use a simple checkout, which could be located just "somewhere". You cannot point to a location outside of the current istallation, I guess changing the PREFIX_PATH in rock is a bad idea in general.

I never said that it is completely broken. Both have their "plus", but I fear that if someone is not so familiar with paths, that person could try to change the prefix instead of the webapp path and "break everything" rather than just use another webapp. Normally you don't have to care about paths at all when using rock, so a lot of users might be unfamiliar with path settings.